### PR TITLE
feature/ads-1054: Use mask for icons in primary nav

### PR DESCRIPTION
**Pull request by APJohns**

### CODE REVIEW

File: public/styles/components/nav-primary/_nav-primary-bar.scss [hunk 1]
```diff
@@ -94,8 +94,8 @@
             /* stylelint-disable scss/selector-nest-combinators */
             &::after,
             &[aria-expanded='true']::after {
-              /* stylelint-enable scss/selector-nest-combinators */
-              background-image: url('../../../assets/images/icon-search.svg');
+              mask-image: url('../../../assets/images/icon-search.svg');
```
Comment (kaseyllabelle) [inline public/styles/components/nav-primary/_nav-primary-bar.scss:97]: can we check browser support? other places have 
```
/* stylelint-disable-next-line property-no-vendor-prefix */
      -webkit-mask: url('../../assets/images/icon-chevron-down.svg') no-repeat 50% 50%;
      mask: url('../../assets/images/icon-chevron-down.svg') no-repeat 50% 50%;
```

Comment (APJohns) [reply public/styles/components/nav-primary/_nav-primary-bar.scss:97]: Seems pretty well supported https://caniuse.com/css-masks

Comment (misdreavus79) [reply public/styles/components/nav-primary/_nav-primary-bar.scss:97]: https://caniuse.com/mdn-css_properties_mask-image_svg_masks

Comment (kaseyllabelle) [reply public/styles/components/nav-primary/_nav-primary-bar.scss:97]: should we align elsewhere then?

File: public/styles/components/nav-primary/_nav-primary-bar.scss [hunk 2]
```diff
@@ -130,9 +132,8 @@
                 right: var(--space-2x);
                 width: $icon-size;
                 height: $icon-size;
-                background-image: url('../../../assets/images/icon-chevron-down.svg');
-                background-position: bottom;
```
Comment (kaseyllabelle) [inline public/styles/components/nav-primary/_nav-primary-bar.scss:134]: do we need to set position?

File: public/styles/components/nav-primary/_nav-primary-menu.scss [hunk 3]
```diff
@@ -40,10 +40,8 @@
             right: 0;
             width: var(--icon-size-sm);
             height: var(--icon-size-sm);
-            background-image: url('../../../assets/images/icon-chevron-right.svg');
-            background-position: center;
-            background-repeat: no-repeat;
-            background-size: cover;
```
Comment (kaseyllabelle) [inline public/styles/components/nav-primary/_nav-primary-menu.scss:46]: did we check varying viewport widths and zoom levels to be sure we don't need to set any of these?

**Review by kaseyllabelle — APPROVED** [review]

**Pull request merged**